jobserver/views/job_requests.py

```diff
@@ -112,6 +117,23 @@ def dispatch(self, request, *args, **kwargs):
             return self.render_to_response(context=context)
 
         self.actions = list(get_actions(data))
+
+        # Find the status of the codelists in this workspace and branch
+        # Codelist status is either "ok" or "error"
+        self.codelists_status = get_codelists_status(
+            self.workspace.repo.owner,
+            self.workspace.repo.name,
+            ref,
+        )
```

Comment:
Should this go in the try/except above so we can handle the exceptions `get_codelists_status` can raise?